src/collections/_documentation/integrations/bitbucket.md

```diff
@@ -1,23 +1,51 @@
 ---
 title: Bitbucket
-sidebar_order: 1
+sidebar_order: 2
```

Comment:
You have two items marked as `2`. This won't break things, but it will cause potentially unexpected ordering confusion since it falls back to alphabetical sorting for duplicates